plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/AltinityQueryRunner.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.clickhouse;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+import io.trino.Session;
+import io.trino.plugin.tpch.TpchPlugin;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.tpch.TpchTable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.trino.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static io.trino.testing.QueryAssertions.copyTpchTables;
+import static io.trino.testing.TestingSession.testSessionBuilder;
+
+public final class AltinityQueryRunner
+{
+    public static final String TPCH_SCHEMA = "tpch";
+
+    private AltinityQueryRunner() {}
+
+    public static DistributedQueryRunner createAltinityQueryRunner(
+            TestingAltinityServer server,
```

Comment:
This is probably the only difference between ClickHouseQueryRunner and this class.
We don't need that duplication, as all we need is a) jdbc url and b) ability to create tpch schema.

BTW Can `server.execute("CREATE DATABASE tpch"`) can be done via the connector now?

Reply:
Changed to `queryRunner.execute("CREATE SCHEMA " + TPCH_SCHEMA);`